lib/rules/require-quoted-keys-in-objects.js

```diff
@@ -44,7 +44,7 @@ module.exports.prototype = {
     check: function(file, errors) {
         file.iterateNodesByType('ObjectExpression', function(node) {
             node.properties.forEach(function(prop) {
-                if (prop.shorthand || prop.method || prop.kind !== 'init') {
+                if (prop.shorthand || prop.method || prop.kind !== 'init' || prop._babelType === 'SpreadProperty') {
```

Comment:
Ok what do you think it should be called? `utils.getBabelType`, `utils.getType`?